### PR TITLE
Rmp story2 filter

### PR DESCRIPTION
**Pull request by alecorra**

### CODE REVIEW

File: public/js/cohortFilter.js [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+function applyCohortFilter(cohortDate) {
+    let mismatches = [];
+    applicantsDates.forEach(function (date, index) {
+        let theDates = date.getAttribute('data-id')
+        if(cohortDate !== theDates) {
+            let row = date.parentNode
+            mismatches.push(row)
+        }
+    })
+    mismatches.forEach(function(mismatch) {
```
Comment (mporam) [inline public/js/cohortFilter.js]: why are you looping over matches? you already have the row on line 15

File: public/js/cohortFilter.js [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+ *
+ * @param cohortDate
+ */
+function applyCohortFilter(cohortDate) {
+    let mismatches = [];
+    applicantsDates.forEach(function (date, index) {
+        let theDates = date.getAttribute('data-id')
+        if(cohortDate !== theDates) {
+            let row = date.parentNode
+            mismatches.push(row)
```
Comment (mporam) [inline public/js/cohortFilter.js]: what is mismatches for? just do `row.classList.add('hidden')`

File: src/Classes/Entities/ApplicantEntity.php [hunk 3]
```diff
@@ -268,4 +290,15 @@ public function getCohortDate()
     {
         return date("F, Y", strtotime($this->cohortDate));
     }
+
+    /**
+     * Gets cohort date as a yyyymmdd
+     * @return false|string
+     */
+    public function getCohortDateNumber()
```
Comment (mporam) [inline src/Classes/Entities/ApplicantEntity.php]: poorly named function, can we follow the same convention as the other date methods and put the format on the method name

File: templates/displayApplicants.phtml [hunk 4]
```diff
@@ -4,9 +4,11 @@
     <meta charset="UTF-8" name="viewport" content="width=device-width, initial-scale=1">
     <title>Registered Applicants</title>
     <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css"
-          integrity="sha384-BVYiiSIFeK1dGmJRAkycuHAHRg32OmUcww7on3RYdg4Va+PmSTsz/K68vbdEjh4u" crossorigin="anonymous">
+          integrity="sha384-BVYiiSIFeK1dGmJRAkycuHAHRg32OmUcww7on3RYdg4Va+PmSTsz/K68vbdEjh4u" crossorigin="anonymous"/>
+    <link rel="stylesheet" type="text/css" href="https://cdn.jsdelivr.net/npm/daterangepicker/daterangepicker.css"/>
+    <link rel="stylesheet" type="text/css" href="/css/genericStyles.css"/>
     <link rel="stylesheet" type="text/css" href="/css/displayApplicants.css">
-    <link rel="stylesheet" type="text/css" href="/css/genericStyles.css">
+<!--    <link rel="stylesheet" type="text/css" href="/css/genericStyles.css">-->
```
Comment (mporam) [inline templates/displayApplicants.phtml]: please remove

File: public/js/cohortFilter.js (hunk 1 above)
Comment (alecorra) [reply public/js/cohortFilter.js]: okay got it :D

File: public/js/cohortFilter.js (hunk 2 above)
Comment (alecorra) [reply public/js/cohortFilter.js]: 👍

**Pull request closed without merging**